public/locales/en/loginDialog.json

```diff
@@ -0,0 +1,7 @@
+{
+  "welcome": "Welcome.",
+  "providesScholarships": "DreamScholars provides scholarships for all students regardless of status.",
+  "joinCommunity": "Join our community &",
+  "getAccess": "Get access to scholarship submissions and other cool features soon.",
+  "signIn": "Sign In"
```

Comment:
This is one of those actions that also appears elsewhere in the site. Maybe save underneath `commons:actions.signIn`?

Reply:
but this text only gets used in the `loginDialog` component no?